css/header.css

```diff
@@ -0,0 +1,193 @@
+ .material-black {
+     color: black
+ }
+
+/* ------- */
+/* Header */
+/* ----- */
+
+.header {
+    grid-template-columns: 1fr;
+}
+
+/* ------ */
+/* Navbar */
+/* ------ */
+
+.navbar {
+    background-color: var(--surface2);
+    min-block-size: 4rem;
+    border-block-end: 1px solid var(--surface4);
+}
+
+.navbar__wrapper {
+    align-items: center;
+    inline-size: min(90%, 1000px);
+    margin-inline: auto;
+    block-size: 100%;
+}
+
+.navbar__list {
+    align-items: center;
+    block-size: 100%;
+}
+
+.navbar__logo {
+    font-size: 1.5rem;
+    color: var(--brand);
+    letter-spacing: 1.5px;
+}
+
+.navbar__list--icons {
+    display: flex;
+    margin-inline-start: auto;
+}
+
+.navbar__list--normal {
+    display: none;
+    gap: 0;
+    margin-inline: auto;
+}
+
+.navbar__list--normal > .navbar__items {
+    letter-spacing: 1.2px;
+}
+
+.navbar__items--name {
+    padding-inline: 1.5rem;
+    padding-block: 1.5rem;
+}
+
+.navbar__items--name:hover {
+    background-color: var(--surface3);
+}
+
+
+/* ------------ */
+/* Hero Section */
+/* ------------ */
+
+.hero {
+    color: var(--surface2);
+}
+
+.hero__wrapper {
+    inline-size: 100%;
+    margin-inline: auto;
+    background-color: var(--brand);
+    flex-direction: column;
+    gap: 3rem;
+    padding-inline: 1rem;
+    padding-block-start: 3rem;
+}
+
+.hero__description {
+    flex-direction: column;
+    gap: 1rem;
+    text-align: center;
+}
+
+.hero__action {
+    align-self: center;
+}
+
+.hero__heading {
+    line-height: 1.2;
+    font-size: var(--fs-700);
+    letter-spacing: 1.2px;
+    max-inline-size: 30ch;
+    margin-inline: auto;
+}
+
+.hero__text {
+    font-size: 1.2rem;
+    line-height: 1.6;
+    max-inline-size: 30ch;
+    margin-inline: auto;
+}
+
+/* ------------- */
+/* Media Queries */
+/* ------------- */
+
+@media (max-width: 299px) {
+    .hero__action {
+        block-size: 40px;
+    }
+
+    .material-black {
+        font-size: 1.5rem !important;
```

Comment:
Try not use use `!important` in your code.

Reply:
It was a material icon so I have to use important otherwise it's not working.